app/src/main/java/com/babylon/wallet/android/presentation/onboarding/restore/mnemonics/RestoreMnemonicsScreen.kt

```diff
@@ -173,24 +173,51 @@ private fun RestoreMnemonicsContent(
                     }
                 )
             } else {
-                RadixPrimaryButton(
+                Column(
                     modifier = Modifier
                         .fillMaxWidth()
-                        .imePadding()
-                        .padding(RadixTheme.dimensions.paddingDefault),
-                    text = stringResource(
-                        when (state.screenType) {
-                            RestoreMnemonicsViewModel.State.ScreenType.Entities -> R.string.recoverSeedPhrase_enterButton
-                            RestoreMnemonicsViewModel.State.ScreenType.SeedPhrase -> R.string.common_continue
-                            RestoreMnemonicsViewModel.State.ScreenType.NoMainSeedPhrase ->
-                                R.string.recoverSeedPhrase_skipMainSeedPhraseButton
+                        .padding(top = RadixTheme.dimensions.paddingMedium)
+                ) {
+                    if (state.screenType is RestoreMnemonicsViewModel.State.ScreenType.Entities) {
+                        if (!state.isMainBabylonSeedPhrase) {
+                            RadixTextButton(
+                                modifier = Modifier
+                                    .fillMaxWidth()
```

Comment:
So for this one you will have paddingLarge + paddingMedium

Reply:
paddingLarge is horizontal and I added vertical (top)